main.py

```diff
@@ -0,0 +1,83 @@
+import os
+from typing import Any, Dict
+
+from dotenv import load_dotenv
+from fastapi import FastAPI, HTTPException
+from openai import OpenAI
+from pydantic import BaseModel
+
+from pseudoanonymize.anonymization import Anonymizer
+from pseudoanonymize.deanonymization import Deanonymizer
+from pseudoanonymize.exceptions import MaxRetriesExceededException
+
+app = FastAPI()
+
+load_dotenv()
+openai_api_key = os.getenv("OPENAI_API_KEY")
```

Comment:
Can you add a file in the root of the repo called `example.env` and specify which env vars are required to run the service but **don't include their values!**.

Example:

```
OPENAI_API_KEY=X
```

Then in the instructions in the README for development setup add an instruction to copy the contents of `example.env` into a file called `.env` and populate the redacted values.

This is just a really helpful practice for other developers so they don't have to waste time figuring out from the code what are all the env vars they need in order to run the service.

Reply:
Done